api/src/main/java/org/pmiops/workbench/cdr/CdrVersionContext.java

```diff
@@ -24,7 +26,12 @@ public static void clearCdrVersion() {
     cdrVersion.remove();
   }
 
+  @NotNull
   public static DbCdrVersion getCdrVersion() {
-    return cdrVersion.get();
+    DbCdrVersion version = cdrVersion.get();
+    if (version == null) {
```

Comment:
I think this is fine, but would be sure to test all endpoints that may call a dao in the `org.pmiops.workbench.cdr.dao` package. If any endpoints were relying on the default version or just forgot to set the `CdrVersionContext`, then we would introduce a regression with this `ServerErrorException`